tests/integration/plots/test_json.py

```diff
@@ -235,12 +235,22 @@ def test_repo_with_plots(
         split_json_result,
     )
     verify_vega_props("confusion.json", json_result, **confusion_props)
+
+
+@pytest.mark.vscode
+def test_repo_with_removed_plots(tmp_dir, capsys, repo_with_plots):
     from dvc.utils.fs import remove
 
+    next(repo_with_plots())
+
     # even if there is no data, call should be successful
     remove(tmp_dir / ".dvc" / "cache")
     remove("linear.json")
     remove("confusion.json")
     remove("image.png")
-    call(capsys)
-    call(capsys, subcommand="diff")
```

Comment:
@pared we only checked that this didn't raise an error but not the expected result. 

I updated `to_json` to return an empty list for vega plots to match the image plots behavior. 

Before https://github.com/iterative/dvc-render/pull/24 , this tests was passing but the returns for vega plots were a vega template without `data`. 

If this is not the expected return format, we need to remove:

https://github.com/iterative/dvc-render/blob/369396f9e833e9b82724b21d0c5f1b853b156d06/src/dvc_render/vega.py#L46-L47

In order to get back to previous behavior